internal/config/loader.go

```diff
@@ -45,12 +45,20 @@ func LoadConfig(useRegistry string, profile string, confDir string) (configurati
 	var registryMsg string
 
 	e := NewEnvironment()
-	if useRegistry != "" {
+	if useRegistry == "" {
+		registryMsg = "Load configuration from local file and bypassing registration in Registry..."
+		configuration, _, err = loadConfigFromFile(profile, confDir)
+	} else {
+		fmt.Fprintf(os.Stdout, "Source of registry url: %s\n", useRegistry)
 		configuration = &common.Config{}
-		useRegistry = e.OverrideUseRegistryFromEnvironment(useRegistry)
-		err = parseRegistryPath(useRegistry, configuration)
-		if err != nil {
-			return
+		if useRegistry == common.RegistryDefault {
```

Comment:
This is for `--registry - Load the url from configuration.toml` , correct?
Environment variable trumps all, so call to OverrideUseRegistryFromEnvironment should occur before check for `== common.RegistryDefault`

Reply:
Thanks for the reminder, I modified to make it call `OverrideUseRegistryFromEnvironment` before checking to load from file or given url .

Reply:
`OverrideUseRegistryFromEnvironment` should happen before `if useRegistry == "" `